boards/launchxl/src/io.rs

```diff
@@ -0,0 +1,36 @@
+use core::fmt::Arguments;
+use kernel::hil::gpio::Pin;
+use cc26xx;
+
+#[cfg(not(test))]
+#[lang = "panic_fmt"]
+#[no_mangle]
+pub unsafe extern "C" fn rust_begin_unwind(
+    _args: Arguments,
+    _file: &'static str,
+    _line: usize,
+) -> ! {
+    let led0 = &cc26xx::gpio::PORT[6]; // Red led
+    let led1 = &cc26xx::gpio::PORT[7]; // Green led
```

Comment:
Should we establish a convention that on panic only the red LED blinks? That would keep it consistent across boards, and make it easier to help with debugging.

Reply:
Also, use the ```new``` [kernel panic routine](https://github.com/helena-project/tock/blob/master/kernel/src/debug.rs#L53-#L66) that @ppannuto implemented recently. All other ```tock-boards``` is using it now I think (or should)

Reply:
FYI, nrf5x have only green LEDs

Reply:
Maybe include a section in the readme's regarding what LED will be blinking if it panics?